### ISSUE TITLE
Timed Out error on any second or more connection attempts

### ISSUE THREAD

**Issue by ckhawks** [txnimc/KryptonReforged#2]
First connection to the server pretty much always works, but any subsequent connections fail with a "Timed Out" status.

Forge version: 36.2.26
KryptonReforged: kryptonreforged-mc1.16.5_v1.0.0.jar
server log: 
[latest.log](https://github.com/TeamDeusVult/KryptonReforged/files/8038314/latest.log)

client log:
[latest.log](https://github.com/TeamDeusVult/KryptonReforged/files/8038315/latest.log)
 
The only mod I had enabled was KryptonReforged.

**Comment by Lordmau5** [comment]
Getting the exact same issue, thought it was another mod at first but tracked it down to this one.

The way I found out about it was because I have the Discord Integration mod installed and set it to kick me if I don't have my Minecraft account linked to my Discord (in my server)

I connected once fine and was kicked with the prompt.
Then, after linking, I was stuck in the "Connecting to server..." screen for around 5 minutes.

Removed the Discord Integration mod because I thought it was causing the issues and tried the same:
Connected once perfectly fine to my server, disconnected, and when I tried connecting again it ran into the "Connecting to server..." timeout again.

**Comment by aking4125** [comment]
Can also confirm happening here as well. First login is always successful, 2nd time "Connecting to server..." forever.

**Comment by Gbergz** [comment]
Can confirm this issue aswell.

**Comment by VenomiXeD** [comment]
Yes, can also confirm this. Running a 1.18.2 and it appears that similar issue occurs. Even the "Pinging server" has some issues.

**Comment by MercilessArchitect** [comment]
can also confirm this. 1.16. on a custom pack.
